README.md

```diff
@@ -12,7 +12,38 @@ Kyma CLI requires the following software:
 
 ## Installation
 
-For the installation instructions, see the [release page](https://github.com/kyma-project/cli/releases).
+Use the following commands to install the Kyma CLI from the latest release. To install a different release change the path in the first command to point to the desired version. For example: 
+
+`curl -Lo kyma.tar.gz https://github.com/kyma-project/cli/releases/download/1.2.0/kyma_Darwin_i386.tar.gz`
```

Comment:
Just a really small one change: ` to ``` like this:

```
curl -Lo kyma.tar.gz https://github.com/kyma-project/cli/releases/download/1.2.0/kyma_Darwin_i386.tar.gz
```

Reply:
Done